docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go

```diff
@@ -144,7 +144,7 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		root object.  Instead, you should reconstruct it every run.  That's what we'll
 		do here.
 
-		We can check if a job is "finished" and whether it succeeded or failed using status
+		We can check if a job is "finished" and whether it Complete or Failed using status
```

Comment:
small grammatical nit:
```suggestion
		We can check if a job is finished and whether it has state "Complete" or "Failed" using status
```

Reply:
right

Reply:
@daniel-hutao shows that it is pushing the nit changes.